lib/user/getPremium.js

```diff
@@ -0,0 +1,51 @@
+// Includes
+const http = require('../util/http.js').func
+
+// Args
+exports.required = ['userId']
+exports.optional = ['jar']
+
+// Docs
+/**
+ * Gets whether or not a user has premium.
+ * @category User
+ * @alias getPremium
+ * @param {number} userId - The id of the user.
+ * @returns {Promise<void>}
```

Comment:
Return type should be `boolean` instead of `void`.
```suggestion
 * @returns {Promise<boolean>}
```